### DISCUSSION TITLE
Can I use this app on linux intel?

### DISCUSSION THREAD

**Discussion by Gutierre19** [unknownskl/xbox-xcloud-client#755] [Q&A]
Hello, I'm about to install linux on my system and I saw that on the downloads page there's a .deb file but the file says "AMD64". I Have this doubt. This file is only for AMD or I can use on Intel too? Because I use an Intel Processor

**Comment by unknownskl** [comment]
AMD64 just means the software is 64bit. It should be working fine on a Intel processor.
